dev-docs/source/DataFilesForTesting.rst

```diff
@@ -131,6 +132,22 @@ This does the following:
 -  tells the user to upload the file(s),
    ``d6948514d78db7fe251efb6cce4a9b83``, to the `remote store <https://testdata.mantidproject.org/ftp/external-data/upload>`_
 
+2: Using the python script addTestData.py in the Testing/SystemTest/scripts directory. It can be called a follows:
```

Comment:
Typo at the end of this sentence

Reply:
Maybe we should also mention that you need to be in the same directory as the `addTestData.py` script to call it this way